### ISSUE TITLE
Not able to un-hide an already hidden element (IE11)

### ISSUE THREAD

**Issue by binarykitchen** [npm-dom/hidden#3]
On IE11, when I do following on an element that already has display set to none

```
hidden(notifyElement, false)
```

then it won't un-hide (aka show) the element.

**Comment by TehShrike** [comment]
Can you run the tests?  If they pass, can you add a new test that fails in the case you're talking about?

If you have any issues running the tests on Windows, let me know.

**Comment by binarykitchen** [comment]
I ll try tomorrow. Won't be easy. For reference, it's this one line
https://github.com/binarykitchen/videomail-client/blob/master/src/wrappers/visuals/notifier.js#L109

where IE11 won't make the element appear again.

**Comment by TehShrike** [comment]
Hopefully it won't be too hard - the tests are supposed to be able to run in any browser on any OS.

```
npm install
npm test
```

should fire up the tests locally, and you can edit the `test` script in the package.json to be `browserify test.js | tape-run --browser=ie` to run the tests in Internet Explorer.

**Comment by binarykitchen** [comment]
![screen shot 2017-01-11 at 14 14 04](https://cloud.githubusercontent.com/assets/177641/21831395/4e55d180-d808-11e6-8112-8db9ee154467.png)

Yes, your tests fail on IE11, see screenshot.

**Comment by TehShrike** [comment]
Good to know! Nice to have existing tests.

If you want to take a stab at fixing, that would be cool, otherwise I might get to it at some point.

**Comment by binarykitchen** [comment]
yeah ... when could you fix these? that bug is breaking my production site and also dont have time at the same time :/

**Comment by TehShrike** [comment]
lol.  Sorry, I'm not using this library in production, and my development isn't funded by any donors, it's just one of the projects I give some help to occasionally.

I don't know what the fix is any more than you do right now.  You're in a better place to fix it than me, since you have the tests running and failing locally in IE11 on your machine.

This library is only a [few lines](https://github.com/npm-dom/hidden/blob/master/index.js#L3-L9) long, so the difficulty isn't in code spelunking, but identifying the IE compatibility issue.

If it's breaking your site in production, I definitely wouldn't wait on my free time.  If you come up with a fix, a pull request would be appreciated.

**Comment by binarykitchen** [comment]
yeah i know ...

... tell me, what's the logic behind `element.style.display = (value || null) && 'none'`? ... how is this supposed to be able to un-hide an element?

**Comment by binarykitchen** [comment]
alright, took my time to do it then ... here a PR

**Comment by TehShrike** [comment]
Fixed by #4.  Published as 1.1.1
